src/ra_server_proc.erl

```diff
@@ -1188,6 +1230,81 @@ handle_effect(RaftState, {aux, Cmd}, EventType, State0, Actions0) ->
         handle_effects(RaftState, Effects, EventType,
                        State0#state{server_state = ServerState}),
     {State, Actions0 ++ Actions};
+handle_effect(_RaftState, {add_member, Conf, ServerId, Members}, _EventType,
```

Comment:
`_RaftState` should be `leader` here to ensure only leaders evaluate this.

Reply:
Done